database/postgres/postgres.go

```diff
@@ -90,6 +93,10 @@ func WithInstance(instance *sql.DB, config *Config) (database.Driver, error) {
 		config.MigrationsTable = DefaultMigrationsTable
 	}
 
+	if config.StatementTimeout == 0 {
```

Comment:
If there is anyone who doesn't want a statement time out or they have been working by adding this too the SQL files they have no option now?

Wdyt about saying that < 0 will not add a context with cancel

Reply:
Or an additional flag to enable timeouts?

Reply:
Let's stick with `== 0`. An additional flag will make the API harder to use.
Since the struct will default to the empty value, `== 0` is a backwards compatible change.

Reply:
can do just pushed something but I can revert it

Reply:
To be explicit, don't use an extra flag to enable/disable statement timeouts, just use a statement timeout `== 0`. Don't bother with negative timeouts for now.

Reply:
love it. if you have a default false flag to use timeouts, then you are not breaking the API and you are adding functionality. 💯

Reply:
TIL: `context.withTimeout()` with a negative timeout behaves like the parent `context`. In this case, if someone enters a negative statement timeout, it will simply run to completion in the background without any timeout.

Reply:
It looks like a negative timeout will result in an immediate cancel (the expected behavior)
https://golang.org/src/context/context.go

Reply:
yup this is how I select context which is consistent with above. I manually tested and can confirm the behavior.
```
if statementTimeout == 0 {
    return context.Background(), nil
}
return context.WithTimeout(context.Background(), statementTimeout)
```